xml/System.Security.Cryptography.X509Certificates/X509Certificate2.xml

```diff
@@ -2743,12 +2743,46 @@ The certificate is encoded according to the IETF RFC 7468 "strict"
         <Parameter Name="allowCommonName" Type="System.Boolean" Index="2" FrameworkAlternate="net-7.0" />
       </Parameters>
       <Docs>
-        <param name="hostname">To be added.</param>
-        <param name="allowWildcards">To be added.</param>
-        <param name="allowCommonName">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="hostname">The host name to match against.</param>
+        <param name="allowWildcards">
+          <see langword="true" /> to allow wildcard matching for <c>dNSName</c> values in the Subject Alternative Name extension; otherwise, <see langword="false" />.</param>
+        <param name="allowCommonName">
+          <see langword="true" /> to allow matching against the subject Common Name value; otherwise, <see langword="false" />.</param>
+        <summary>Checks to see if the certificate matches the provided hostname.</summary>
+        <returns>
+          <see langword="true" /> if the certificate is a match for the requested hostname; otherwise, <see langword="false" /></returns>
+        <remarks>
+          <para> This method is a platform neutral implementation of IETF RFC 6125 host matching logic.
```

Comment:
Check formatting.

Reply:
Fixing it, it looks bad.

![image](https://user-images.githubusercontent.com/1175054/191815004-4901c929-b865-496a-8c38-14c7842d28fc.png)